gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.caching=true
+version=0.0.1-SNAPSHOT
```

Comment:
had to define it, otherwise pf4j complained for a unparseable version during plugins startup (seems plugin dependency triggered that parsing, even there is no version specified there)